velox/functions/sparksql/tests/MapTest.cpp

```diff
@@ -105,5 +105,89 @@ TEST_F(MapTest, ErrorCases) {
       {inputVector1, inputVector2, inputVector11, inputVector22},
       true);
 }
+
+TEST_F(MapTest, complexTypes) {
+  auto rowKey = makeRowVector({
+      makeFlatVector<int64_t>(1, [](auto row) { return row; }),
+  });
+  auto rowValue = makeRowVector({
+      makeFlatVector<int64_t>(1, [](auto row) { return row; }),
+  });
+  auto mapRowVector = makeMapVector(
+      {
+          0,
+      },
+      rowKey,
+      rowValue);
+  mapSimple("map(c0, c1)", {rowKey, rowValue}, false, mapRowVector);
+
+  auto arrayKey = makeArrayVector<int64_t>({
```

Comment:
consider makeArrayVectorFromJson

please, test with vectors of size > 1

Reply:
Updated, thanks.